### PR TITLE
fix(metrics): include template name in sms.sent event

### LINKED ISSUE #1839: Add SMS templates to Documentation
SMS events are now documented here:
https://github.com/mozilla/fxa-auth-server/blob/master/docs/metrics-events.md

Events are called:
sms.${templateName}.sent

It is unclear what the template names are.

So I ran this to figure it out:
SELECT *
FROM flow_events
WHERE type LIKE 'sms%'
LIMIT 100

The answer is that templates are numbers. Example: ms.1.sent

Let's document what those templates numbers mean.

### PR DESCRIPTION
**Pull request by philbooth**
Fixes #1839.

@mozilla/fxa-devs r?

### CODE REVIEW

File: docs/metrics-events.md [hunk 1]
```diff
@@ -264,6 +264,11 @@ to each of the table names mentioned above:
 
 ## Significant changes
 
+### Train 86
+
+* [The `sms.${templateName}.sent event
+  was fixed](https://github.com/mozilla/fxa-auth-server/pull/1843).
```
Comment (shane-tomlinson) [inline docs/metrics-events.md]: This message is kinda vague, the PR title seems like it would be a perfect one liner.

File: lib/routes/sms.js [hunk 2]
```diff
@@ -83,11 +86,11 @@ module.exports = (log, config, customs, sms) => {
         }
 
         function sendMessage (senderId) {
-          return sms.send(phoneNumber, senderId, messageId, acceptLanguage)
+          return sms.send(phoneNumber, senderId, templateName, acceptLanguage)
         }
 
         function logSuccess () {
-          return request.emitMetricsEvent(`sms.${messageId}.sent`)
+          return request.emitMetricsEvent(`sms.${templateName}.sent`)
```
Comment (shane-tomlinson) [inline lib/routes/sms.js]: @davismtl asked that we list each of the possible template values in [metrics-events.md](https://github.com/mozilla/fxa-auth-server/blob/master/docs/metrics-events.md), worthwhile to do as a part of this PR?

Comment (philbooth) [inline lib/routes/sms.js]: Good point. I started to do this as part of this PR just now, but it started to get out of hand as I had to move big chunks of the document round, add extra sections, and started documenting things like the view names too. Now I'm thinking I should maybe do all that together in a separate documentation PR?

Comment (shane-tomlinson) [inline lib/routes/sms.js]: Sounds good to me!

**Pull request merged**